### PR TITLE
Nc news12

### PR DESCRIPTION
**Pull request by Ryanlee2690**

### CODE REVIEW

File: __tests__/endpoint.test.js [hunk 1]
```diff
@@ -39,4 +39,33 @@ describe('GET /api/topics', () => {
...
+})
+
+describe('GET /api/articles/:article_id', () => {
+    test('status:200, responds with a single matching articles', () => {
+      const ARTICLE_ID = 2;
+      return request(app)
+        .get(`/api/articles/${ARTICLE_ID}`)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.article).toEqual({
```
Comment (cavalierswain) [inline __tests__/endpoint.test.js:51]: Good to see that you're being specific when testing for this 👍

File: __tests__/endpoint.test.js [hunk 2]
```diff
@@ -39,4 +39,33 @@ describe('GET /api/topics', () => {
...
+            created_at: '2020-10-16T05:03:00.000Z',
+            votes: 0
+          });
+        })
+    });
+    test('status something, put in a number that isnt there', () => {
```
Comment (cavalierswain) [inline __tests__/endpoint.test.js]: Not sure about this test name!

Comment (Ryanlee2690) [reply __tests__/endpoint.test.js]: whoops....

**Comment by cavalierswain** [comment]
Looks good! Great work.

**Comment by P-Copley** [comment]
Looks good now @Ryanlee2690 👍

**Pull request merged**